inference-engine/cmake/config.cmake.in

```diff
@@ -30,5 +30,5 @@ else()
 
     get_target_property(InferenceEngine_INCLUDE_DIRS IE::inference_engine INTERFACE_INCLUDE_DIRECTORIES)
     set(InferenceEngine_LIBRARIES IE::inference_engine_legacy IE::inference_engine
-                                  IE::inference_engine_c_api)
+                                  IE::inference_engine_c_api IE::ngraph)
```

Comment:
looks like this can be left as is, because public dependency between IE and ngraph will exported from `inference-engine/src/inference-engine/CMakeLists.txt`